rmgpy/molecule/inchi.py

```diff
@@ -376,6 +376,8 @@ def _generate_minimum_resonance_isomer(mol):
     Next, we return the candidate with the lowest unpaired electrons metric.
 
     The metric is a sorted list with indices of the atoms that bear an unpaired electron
+
+    This function is currently depreciated
```

Comment:
Spelling of `deprecated`. Also, it would be good to mention why.

I think you can add a deprecation warning similar to f7dd2f18621c8ba6a1e78f688d58b817a1b42d50 for now.